projboard/tests/test_article.py

```diff
@@ -0,0 +1,148 @@
+import pytest
+from projboard.models.article import Article, Like, View, Subject, User
+
+
+TITLE = "Article Test"
+CONTENT = "I'm not gonna write all that!"
+
+
+class TestArticle:
+
+    @pytest.fixture
+    def sup(self):
+        return Subject(name='N')
+
+    @pytest.fixture
+    def sup1(self):
+        return Subject(name='R')
+
+    @pytest.fixture
+    def user(self):
+        return User(name="Anas", nickname="A99", email="Anas@gmail.com", password="Anas123")
+
+    @pytest.fixture
+    def art(self, user, sup):
```

Comment:
Please review fixture methods names
Give names that can be readable once you see them elsewhere

art should be called create_aricle the same for sup-> create_subjects and so on